File: CNC modeling 2.0/src/main/java/com/sergey/pisarev/interfaces/Drawing.java
package com.sergey.pisarev.interfaces;

import com.sergey.pisarev.model.MyData;
import com.sergey.pisarev.model.Point;
import javafx.scene.canvas.GraphicsContext;

public interface Drawing {
     void drawContour(MyData data, GraphicsContext gc, Point pointCoordinateZero, double zoom, int index);
     void setNumberLine(int numberLine);
}

File: CNC modeling 2.0/src/main/java/com/sergey/pisarev/interfaces/Drawing.kt
package com.sergey.pisarev.interfaces

import com.sergey.pisarev.model.core.MyData
import com.sergey.pisarev.contur.Point
import javafx.scene.canvas.GraphicsContext

interface Drawing {
    fun drawContour(data: MyData, gc: GraphicsContext, pointCoordinateZero: Point, zoom: Double, index: Int, isSelectToolRadius: Boolean)
    fun setNumberLine(numberLine: Int)
}